plugins/inputs/ravendb/ravendb.go

```diff
@@ -0,0 +1,644 @@
+		"license_type":                                                  serverResponse.License.Type,
+		"license_utilized_cpu_cores":                                    serverResponse.License.UtilizedCpuCores,
+		"license_max_cores":                                             serverResponse.License.MaxCores,
+		"network_tcp_active_connections":                                serverResponse.Network.TcpActiveConnections,
+		"network_concurrent_requests_count":                             serverResponse.Network.ConcurrentRequestsCount,
+		"network_total_requests":                                        serverResponse.Network.TotalRequests,
+		"network_requests_per_sec":                                      serverResponse.Network.RequestsPerSec,
+		"cluster_node_state":                                            mapNodeState(serverResponse.Cluster.NodeState),
+		"cluster_current_term":                                          serverResponse.Cluster.CurrentTerm,
+		"cluster_index":                                                 serverResponse.Cluster.Index,
+		"databases_total_count":                                         serverResponse.Databases.TotalCount,
+		"databases_loaded_count":                                        serverResponse.Databases.LoadedCount,
+		"cpu_machine_io_wait":                                           serverResponse.Cpu.MachineIoWait,
+		"license_expiration_left_in_sec":                                serverResponse.License.ExpirationLeftInSec,
+		"network_last_request_time_in_sec":                              serverResponse.Network.LastRequestTimeInSec,
+		"network_last_authorized_non_cluster_admin_request_time_in_sec": serverResponse.Network.LastAuthorizedNonClusterAdminRequestTimeInSec,
+		"certificate_server_certificate_expiration_left_in_sec":         serverResponse.Certificate.ServerCertificateExpirationLeftInSec,
+	}
+
+	if serverResponse.Config.TcpServerUrls != nil {
+		fields["config_tcp_server_urls"] = strings.Join(serverResponse.Config.TcpServerUrls, ";")
+	}
+
+	if serverResponse.Config.PublicTcpServerUrls != nil {
+		fields["config_public_tcp_server_urls"] = strings.Join(serverResponse.Config.PublicTcpServerUrls, ";")
+	}
+
+	if serverResponse.Certificate.WellKnownAdminCertificates != nil {
+		fields["certificate_well_known_admin_certificates"] = strings.Join(serverResponse.Certificate.WellKnownAdminCertificates, ";")
+	}
+
+	acc.AddFields("ravendb_server", fields, tags)
+}
+
+func mapMemorySeverity(severity string) int8 {
+	switch severity {
+	case "None":
+		return 0
+	case "Low":
+		return 1
+	case "ExtremelyLow":
+		return 2
+	default:
+		return -1
+	}
+}
+
+func mapNodeState(state string) int8 {
+	switch state {
+	case "Passive":
+		return 0
+	case "Candidate":
+		return 1
+	case "Follower":
+		return 2
+	case "LeaderElect":
+		return 3
+	case "Leader":
+		return 4
+	default:
+		return -1
+	}
+}
+
+func gatherDatabases(r *RavenDB, acc telegraf.Accumulator) {
```

Comment:
Please change to
```suggestion
func (r *RavenDB) gatherDatabases(acc telegraf.Accumulator) {
```
same for all other members of the `gather` family.

Reply:
ok refactored that a bit and remove list of functions.

Reply:
Please mark as resolved.